test/expression/condition_comment.stmt

```diff
@@ -0,0 +1,62 @@
+40 columns                              |
+>>> Inline block comment before `?`.
+cond /* c */ ? 1 : 2;
+<<<
+cond /* c */ ? 1 : 2;
+>>> Inline block comment after `?`.
+cond ? /* c */ 1 : 2;
+<<<
+cond ? /* c */ 1 : 2;
+>>> Inline block comment before `:`.
+cond ? 1 /* c */ : 2;
+<<<
+cond ? 1 /* c */ : 2;
+>>> Inline block comment after `:`.
+cond ? 1 : /* c */ 2;
+<<<
+cond ? 1 : /* c */ 2;
+>>> Inline block comment after else.
+cond ? 1 : 2 /* c */;
+<<<
+cond ? 1 : 2 /* c */;
+>>> Line comment before `?`.
+cond // c
+? 1 : 2;
+<<<
+cond // c
+    ? 1
+    : 2;
+>>> Line comment after `?`.
+cond ? // c
+1 : 2;
+<<<
+### Looks weird, but users generally won't put comments here.
+cond
+    ? // c
+    1
+    : 2;
+>>> Line comment before `:`.
+cond ? 1 // c
+: 2;
+<<<
+cond
+    ? 1 // c
+    : 2;
+>>> Line comment after `:`.
+cond ? 1 : // c
+2;
+<<<
+### Looks weird, but users generally won't put comments here.
+cond
+    ? 1
+    : // c
```

Comment:
Just me being picky, but does this make more sense for the comment to go with the 1 and maybe same with some of the other examples? I guess you don't do comment moving for conditionals?

Reply:
Yeah, I just didn't want to bother moving comments around for conditionals. Usually, it's easiest to just leave the comment where it is and the user will figure it out and move it themselves.

Lists and related constructs get more sophisticated comment handling since they really need it because the formatter may be adding and removing commas.

Eventually, I could see us doing more comment normalization and making stuff nicer like here, but for now I'm trying to not do too many style changes at the same time.